astropy_helpers/commands/_dummy.py

```diff
@@ -61,7 +61,7 @@ def __init__(cls, name, bases, members):
                 "attribute.")
 
     def __getattribute__(cls, attr):
-        if attr in ('description', 'error_msg'):
+        if attr in ('description', 'error_msg') or attr.startswith('_'):
```

Comment:
This makes sense anyway since the custom error is only meant to be shown for command arguments, which should be public.